src/edu/csus/ecs/pc2/core/PacketHandler.java

```diff
@@ -3551,7 +3570,29 @@ private void checkoutRun(Packet packet, Run run, ClientId whoRequestsRunId, bool
 
                         theRun = contest.checkoutRun(run, whoRequestsRunId, false, computerJudge);
 
-                        RunFiles runFiles = contest.getRunFiles(run);
+                        RunFiles runFiles = null;
+                        
+                        // in case GetRunFiles throws an exception we want to deal with it separately
+                        try {
+                            runFiles = contest.getRunFiles(run);  
+                        } catch (Exception e) {
+                            controller.getLog().info("contest.getRunFiles can not get files for run " + run.getNumber() + " (settng to status NEW): " + e.getMessage());
+                            
```

Comment:
As above, I think this is ALWAYS indicative of problem (can't getRunFiles() for some reason) so this should be (at least) a `warning()`, not just `info()`.